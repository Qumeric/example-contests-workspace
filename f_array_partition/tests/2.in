1
10
6 1 4 1 6 7 1 3 6 3