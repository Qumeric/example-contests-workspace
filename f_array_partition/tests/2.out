YES 
4 2 4